R/get_webtext.R

```diff
@@ -0,0 +1,361 @@
+#' Check hospital reporting data latency and completeness.
+#'
+#' This function retrieves hospital reporting data from
+#' data.cdc.gov and checks for data latency and reporting
+#' completeness issues. It returns a flag string
+#' describing any reporting issues.
+#'
+#' @param reference_date Date, the reference date for the
+#' forecast.
+#' @param disease Character, disease name ("covid" or
+#' "rsv").
+#' @param excluded_locations Character vector of location
+#' codes to exclude. Default: character(0).
+#'
+#' @return Character string describing reporting issues,
+#' or empty string if no issues.
+check_hospital_reporting_latency <- function(
+  reference_date,
+  disease,
+  excluded_locations = character(0)
+) {
+  desired_weekendingdate <- as.Date(reference_date) - lubridate::dweeks(1)
+
+  disease_abbr <- dplyr::case_match(
+    disease,
+    "covid" ~ "c19",
+    "rsv" ~ "rsv"
+  )
+
+  reporting_column <- glue::glue(
+    "totalconf{disease_abbr}newadmperchosprepabove80pct"
+  )
+
+  percent_hosp_reporting_below80 <- forecasttools::pull_data_cdc_gov_dataset(
+    dataset = "mpgq-jmmr",
+    columns = c(reporting_column),
```

Comment:
```suggestion
    columns = reporting_column,
```
Not needed because in R a character string is a length-1 character vector!

Reply:
This is one of those places where I think I was performizing pre-mature optimization (keeping `c()` in case additional columns). Thank you for flagging.